components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/response/AccessTokenResponseHandler.java

```diff
@@ -0,0 +1,22 @@
+package org.wso2.carbon.identity.oauth2.token.handlers.response;
```

Comment:
add license header

Reply:
fixed in f2bda44b173e8e1643ee4cc0e2b72b2dd02bead3